src/lib/Log.ml

```diff
@@ -13,12 +13,22 @@ let pp_lvl fmt =
   | `Warn ->
     Format.fprintf fmt "Warn"
 
-let pp_message ~loc ~lvl pp data =
+let pp_message ~loc ~lvl ~last_token pp data =
   match loc with
   | None ->
     pp Format.std_formatter data
   | Some span ->
-    Format.printf "@[<v>%a [%a]:@,  @[<v>%a@]@]@.@."
-      LexingUtil.pp_span span
-      pp_lvl lvl
-      pp data
+    match lvl with
+    | `Info ->
+      Format.printf "@[<v>%a@]@."
```

Comment:
I would prefer to keep the level and span, because it is part of the protocol that allows cooltt to be used in our editor mode; the purpose is to display an indicator near where the information emanated.

Reply:
Understood. I've uploaded a change that preserves the formatting.